iis-wf/iis-wf-primary/src/test/java/eu/dnetlib/iis/wf/primary/processing/ClassPathBasedPatentServiceFacade.java

```diff
@@ -0,0 +1,53 @@
+package eu.dnetlib.iis.wf.primary.processing;
+
+import java.io.IOException;
+import java.util.NoSuchElementException;
+
+import org.apache.commons.io.IOUtils;
+
+import eu.dnetlib.iis.referenceextraction.patent.schemas.ImportedPatent;
+import eu.dnetlib.iis.wf.referenceextraction.patent.PatentServiceException;
+import eu.dnetlib.iis.wf.referenceextraction.patent.PatentServiceFacade;
+
+/**
+ * Simple mock retrieving XML concents as files from classpath. Relies on
```

Comment:
I think this one is still not changed.

Reply:
Again, really strange. It worries me because it may be the case some other changes got lost. I will double check the sources again and check the local history.

Reply:
I think I have found it committed:

https://github.com/openaire/iis/pull/1091/commits/e3c3d1e30abffdff3081aa033b69b0e3613ad21e#diff-f061ab1c7b67b5f05967391a21fbb5ce

Reply:
But this is in a different package. There are two classes named `ClassPathBasedPatentServiceFacade`.

Reply:
I can confirm: fixed in both.